core/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for core project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-+*+^0m=93&53me!gzbdij*cs_wj-)m30$!6w_%i(i$5@*-j6-l'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Security Issue: `DEBUG` is set to `True`, which is not recommended for production environments. Set it to `False` to avoid exposing sensitive information.